### ISSUE TITLE
Goblin Hunter lv 10 class quest breadcrumb leads to old 1.15 durotar instead of new 1.17.1 blackstone island NPC

### ISSUE THREAD

**Issue by remilege** [slowtorta/turtlewow-bug-tracker#5746]
**Please, provide a clear description what the bug is:**

Viz Fizbeast, the NPC giving the hunter quest for taming in Razor Hill, has disappeared. It's not where it should be regarding pfquest and the search I found on turtle discord, next to the blue tent in Razor Hill.
https://database.turtle-wow.org/?npc=80903
I can get the Hunter's path quest correctly in TB, Orgr and Razor Hill  from the Hunter trainers but cant continue.

**Steps to reproduce the behavior with as much detail as possible:**

1. Go to Razor Hill
2. See the missing NPC at the blue tent
3. Grumble 'cause you cant have a pet :)

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=80903
Quest: https://database.turtle-wow.org/?search=hunter%27s+path

**Expected behavior. Describe how it should work:**

The NPC should be right next to the blue tent in Razor Hill.

Merry Christmas if you do it or Happy Holidays if you dont !
Thanks in advance.

**Comment by twowTurtReynolds** [comment]
Looks like the 1.15.0 NPC was replaced in 1.17.1 with this questline: https://database.turtle-wow.org/?quest=60160

**Comment by remilege** [comment]
Thank you.
Still, as a goblin, you get the original quest "taming the beast" from the Orgrimmar and Thunder Bluff Hunter trainer to go and see Viz Fizbeast in Razor Hill and pfquest having the same quest objective. Which is very misleading.
Furthermore, the quest you mention is buggued and the taming rod doesnt workhttps://database.turtle-wow.org/?quest=60160I already opened another bug ticket for this issue. I'm level 22 and still have no pet as a hunter ...

    Le lundi 25 décembre 2023 à 01:42:33 UTC+1, Turt Reynolds ***@***.***> a écrit :  
 
 


Looks like the 1.15.0 NPC was replaced in 1.17.1 with this questline: https://database.turtle-wow.org/?quest=60160

**Comment by twowTurtReynolds** [comment]
You are correct, the breadcrumb does need updating. I will reopen this ticket for that.
![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/151180101/15851869-c089-41d1-8aca-6f05804cb63a)

As for the fact you can't get the spells due to https://github.com/slowtorta/turtlewow-bug-tracker/issues/5750 , well there's nothing the QA team can do for you, instead please contact a GM who might be able to help, and reference that you have an open bug ticket for the issue that might not be resolved soon.
